File: javaArchive/src/IDs/LectureID.java
package IDs;

//Samet CAN 150120528
public class LectureID implements UniqueID {

	private String lectureCode;

	public LectureID(String lectureCode) {
		super();
		this.lectureCode = lectureCode;
	}

	// Another set method for ID
	public void setID(String string) {
		this.lectureCode = string;

	}

	public String getID() {
		return lectureCode;
	}

}

File: pythonCode/LectureID.py
class LectureID:
    def __init__(self, __lectureCode: str):
        self.__lectureCode = __lectureCode

     # Creating another properties for variables
    def setID(self, string: str):
        self.__lectureCode = string

    def getID(self) -> str:
        return self.__lectureCode

